resolwe/flow/filters.py

```diff
@@ -14,12 +14,12 @@
 class BaseResolweFilter(filters.FilterSet):
     """Base filter for Resolwe's endpoints."""
 
-    id = filters.AllLookupsFilter()  # pylint: disable=invalid-name
-    slug = filters.AllLookupsFilter()
-    name = filters.AllLookupsFilter()
+    id = filters.AutoFilter(lookups='__all__')  # pylint: disable=invalid-name
```

Comment:
It would be great if you can explicitly list lookups. I know it is quite some work, but it will be a lot easier in the future, if we know which use-cases should be supported (e.g. in Elasticsearch), what should be tested,...

Reply:
While removing DRF-fiters, some of the syntax had to be changed. I tried to limit filter functionality to filter explicitly declared fields/lookups. Tests pass, but there might be cases where desired functionality is not tested. @lukaw3d and @hadalin should advise if there are which cases that are not covered by tests...